src/renderers/WebGLRenderer.js

```diff
@@ -2386,24 +2403,43 @@ class WebGLRenderer {
 			_gl.pixelStorei( _gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, dstTexture.premultiplyAlpha );
 			_gl.pixelStorei( _gl.UNPACK_ALIGNMENT, dstTexture.unpackAlignment );
 
+			const unpackRowLen = _gl.getParameter( _gl.UNPACK_ROW_LENGTH );
+			const unpackImageHeight = _gl.getParameter( _gl.UNPACK_IMAGE_HEIGHT );
+			const unpackSkipPixels = _gl.getParameter( _gl.UNPACK_SKIP_PIXELS );
+			const unpackSkipRows = _gl.getParameter( _gl.UNPACK_SKIP_ROWS );
+			const unpackSkipImages = _gl.getParameter( _gl.UNPACK_SKIP_IMAGES );
+
```

Comment:
So this block is to store existing state values so they can be restored after the copy, right?

If so, can we use the `current*` or `old*` syntax like at other places?

Reply:
I just copied the same lines from `copyTextureToTexture3D` - do you want it changed there, as well? And which is the preferred syntax, `current*` or `old*`? I think we should pick one.

Reply:
Looks like `current*` is used more frequently so I'll update it to that 😁